src/chunk.c

```diff
@@ -849,6 +925,43 @@ append_chunk_oid(ChunkScanCtx *scanctx, Chunk *chunk)
 	return true;
 }
 
+static bool
+is_numeric_type(Oid type)
+{
+	return type == INT8OID || type == INT4OID || type == INT2OID;
+}
+
+static bool
+is_valid_time_column_type(Oid type)
+{
+	return is_numeric_type(type) ||type == InvalidOid || type == DATEOID ||
```

Comment:
why id InvalidOid in here?

Reply:
the idea was that you could skip the argument and it should still typecheck but I found a more clear way to communicate that.